scripts/travis/configure_dev.sh

```diff
@@ -3,60 +3,56 @@
 # keep script execution on errors
 set +e
 
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+function installGo() {
+    OS_ARCH=$1
+    ARCH=$2
+    GO_VERSION=$("${SCRIPTPATH}/../get_golang_version.sh")
+    INSTALLED_GO_VERSION=$("${SCRIPTPATH}/../get_installed_golang_version.sh")
+    echo "Ensure Go version ${GO_VERSION} for arch ${OS_ARCH}"
 
-OS=$("${SCRIPTPATH}/../ostype.sh")
-ARCH=$("${SCRIPTPATH}/../archtype.sh")
-
-if [ "${OS}" = "linux" ]; then
-    if [[ "${ARCH}" = "arm64" ]]; then
-        go version 2>/dev/null
-        if [ "$?" != "0" ]; then
-            echo "Go cannot be found; downloading..."
-            # go is not installed ?
-            wget -q https://dl.google.com/go/go1.12.9.linux-arm64.tar.gz
-            if [ "$?" = "0" ]; then   
+    if [[ "${INSTALLED_GO_VERSION}" != "${GO_VERSION}" ]]; then
+        echo "Installing go version ${GO_VERSION} to replace ${INSTALLED_GO_VERSION}"
+        if [[ "${ARCH}" = "amd64" ]]; then
+            eval "$(gimme ${GO_VERSION})"
```

Comment:
Using gimme here is ok, however, it creates a different execution path for amd64 vs non-amd64.
If you could have "taken" the gimme script and use it in all the cases, it would create a more uniform script.

Reply:
@tsachiherman Updated go version management for Travis builds to use gimme for all platforms.